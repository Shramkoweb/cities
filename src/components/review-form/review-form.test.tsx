import * as React from "react";
import {configure, shallow} from "enzyme";
import * as Adapter from "enzyme-adapter-react-16";import {ReviewForm} from './review-form';

configure({adapter: new Adapter()});

it(`Review form renders correctly`, () => {
  const reviewForm = shallow(<ReviewForm
    onInputChange={jest.fn()}
    rating={`5`}
    review={`Some test`}
    isReviewSent={true}
    onUpdateForm={jest.fn()}
    isReviewSending={true}
    id={5}
    onSendForm={jest.fn()}
    onFormReset={jest.fn()}
    isValid={true}
  />);

  expect(reviewForm).toMatchSnapshot();
});
