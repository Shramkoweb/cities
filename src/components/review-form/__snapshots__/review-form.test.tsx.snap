// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review form renders correctly 1`] = `ShallowWrapper {}`;
